docs/OV_Runtime_UG/model_representation.md

```diff
@@ -32,15 +32,15 @@ OpenVINO™ Runtime allows to use different approaches to work with model inputs
 
 @endsphinxtabset
 
-OpenVINO™ Runtime model representation uses special classes to work with model data types and shapes. For data types the `ov::element::Type` is used.
+OpenVINO™ Runtime model representation uses special classes to work with model data types and shapes. The `ov::element::Type` is used for data types.
 
 ## Shapes Representation
 
 OpenVINO™ Runtime provides two types for shape representation: 
 
-* `ov::Shape` - Represents static (fully defined) shapes.
+* The `ov::Shape` - Represents fully defined (static) shapes.
 
-* `ov::PartialShape` - Represents dynamic shapes. That means that the rank or some of dimensions are dynamic (dimension defines an interval or undefined). `ov::PartialShape` can be converted to `ov::Shape` using the `get_shape()` method if all dimensions are static; otherwise the conversion raises an exception.
+* The `ov::PartialShape` - Represents dynamic shapes. That means that the rank or some of dimensions are dynamic (dimension defines an interval or undefined). The `ov::PartialShape` can be converted to the `ov::Shape`, using the `get_shape()` method if all dimensions are static. Otherwise, the conversion throws an exception.
```

Comment:
```suggestion
* `ov::PartialShape` - Represents dynamic shapes. This means that the rank or some of dimensions are dynamic (dimension defines an interval or undefined). `ov::PartialShape` can be converted to `ov::Shape`, using the `get_shape()` method if all dimensions are static. Otherwise, the conversion throws an exception.
```